packages/react-ui/package.json

```diff
@@ -73,6 +74,7 @@
     "@coopdigital/component-notification": "^1.2.0",
     "@coopdigital/css-foundations": "^7.1.2",
     "@coopdigital/foundations-buttons": "^2.5.7",
+    "@coopdigital/foundations-forms": "^5.3.2",
```

Comment:
strip foundation-forms out.  
css-foundations will handle this

Reply:
I'll strip out foundation-forms on next feature branch, after current PRs are merged.